blocks/library/embed/index.js

```diff
@@ -21,57 +23,141 @@ registerBlock( 'core/embed', {
 	category: 'common',
 
 	attributes: {
-		url: attr( 'iframe', 'src' ),
 		title: attr( 'iframe', 'title' ),
 		caption: children( 'figcaption' ),
 	},
 
-	edit( { attributes, setAttributes, focus, setFocus } ) {
-		const { url, title, caption } = attributes;
+	edit: class extends wp.element.Component {
+		constructor() {
+			super( ...arguments );
+			this.doServerSideRender = this.doServerSideRender.bind( this );
+			// Copies the block's url so we can edit it without having the block update
+			this.state = {
+				oEmbedHtml: '',
+				error: false,
+				fetching: false,
+			};
+			this.noPreview = [
+				'facebook.com',
+			];
+			this.videoEmbeds = [
```

Comment:
That's fantastic, thank you for letting me know about this! I'll work on changing this over.